src/main/java/seedu/duke/Duke.java

```diff
@@ -33,16 +37,22 @@ private static void start() {
 
     private static void run() {
         Boolean isExit = false;
+        Scanner in = new Scanner(System.in);
 
         //loop
         while (!isExit) {
             //scan
-
+            String input = in.nextLine();
             //parse
-            Command command = parser.parse(input);
+            Command command = null;
```

Comment:
Actually is this needed, since the variable command is only used in the try block?